css/style.css

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Avoid leaving unnecessary blank spaces. This may distort readability.

Reply:
- [ ] Kindly use comments politely, as suggestions instead of orders.

Reply:
Noted thank you. As below?

You may try not to leave blank spaces as this may distort readability.